tests/test_sequence_packer.py

```diff
@@ -39,20 +39,21 @@ def packing_config(packing_style: PackingStyleType, overflow_type: OverflowType)
 
 
 @pytest.fixture
-def sequence_packer(max_seq_length: int, eos_token_id: int, packing_config: PackingConfig) -> SequencePacker:
+def sequence_packer(max_seq_length: int, pad_token_id: int, packing_config: PackingConfig) -> SequencePacker:
     """Create the sequence packer."""
-    return SequencePacker(max_seq_length, eos_token_id, packing_config, Metrics())
+    return SequencePacker(max_seq_length, pad_token_id, packing_config, Metrics())
 
 
 @pytest.mark.parametrize(
-    "length,max_seq_length,eos_token_id,packing_style,overflow_type,length_2,max_seq_length_2,eos_token_id_2,\
+    "length,max_seq_length,pad_token_id,eos_token_id,packing_style,overflow_type,length_2,max_seq_length_2,eos_token_id_2,\
```

Comment:
Are `eos_token_id` and `eos_token_id_2` still used here?

Reply:
Yes, they are passed into tokenized_line and tokenized_line_2 fixtures 

def test_handle_overflow(
    sequence_packer: SequencePacker,
    tokenized_line: TokenizedLine,
    tokenized_line_2: TokenizedLine,
    expected_token_ids: List[int],
):

@pytest.fixture
def tokenized_line(length: int, max_seq_length: Optional[int], eos_token_id: Optional[int]) -> TokenizedLine:
    """Creates a tokenized line."""
    return get_tokenized_line(length, max_seq_length, eos_token_id)


@pytest.fixture
def tokenized_line_2(length_2, max_seq_length_2, eos_token_id_2):
    """Creates a tokenized line.  To be used if you've already used the other tokenized line fixture
    in your test function.
    """
    return get_tokenized_line(length_2, max_seq_length_2, eos_token_id_2)